addons/website_slides/tests/test_attendee.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website_slides.tests import common
+
+
+class TestAttendee(common.SlidesCase):
+
+    def test_course_atttendee(self):
+        self.channel.write({'partner_ids': [(4, self.customer.id)]})
+        attendee_before = self.env['slide.channel.partner'].search_count(
+            ['&', ('channel_id', '=', self.channel.id), ('partner_email', '=', self.customer.email)]
+        )
+        self.customer.copy()
+        attendee_after = self.env['slide.channel.partner'].search_count(
```

Comment:
We could also check that global number of attendees did not change at all (not only linked to customer email).